src/Calendar.jsx

```diff
@@ -441,7 +450,7 @@ export default class Calendar extends Component {
             onClick={mergeFunctions(clickAction, onClickDay)}
             onClickWeekNumber={onClickWeekNumber}
             onMouseLeave={onMouseLeave}
-            showFixedNumberOfWeeks={showFixedNumberOfWeeks}
+            showFixedNumberOfWeeks={showFixedNumberOfWeeks || showDoubleView}
```

Comment:
@wojtekmaj  thanks for the great lib!
why was `showFixedNumberOfWeeks` forced to `true` in double view?
mind if I PR to remove this?

Reply:
This is because otherwise the UI would look quite bad with different amount of rows on each side.

Reply:
@wojtekmaj  got it. so it's subjective. ex. priceline.com implements a dual view without locking down # of weeks. 
are you open to a PR to remove this?

Reply:
I like this idea.